docs/docs/concepts.mdx

```diff
@@ -607,6 +607,7 @@ For specifics on how to use callbacks, see the [relevant how-to guides here](/do
 ## Techniques
 
 ### Streaming
+<span data-heading-keywords="stream,streaming"></span>
```

Comment:
nice - TIL
Is this docusaurus/algolia generic or do we have a custom script here?

Reply:
Custom script that elevates specific headers in search results. Use VERY sparingly